corehq/form_processor/tests/test_case_dbaccessor.py

```diff
@@ -563,6 +564,29 @@ def test_case_has_transactions_since_sync(self):
             CaseAccessorSQL.case_has_transactions_since_sync(case1.case_id, "foo", datetime.utcnow())
         )
 
+    def test_get_all_cases_modified_since(self):
+        case1 = _create_case(user_id="user1")
+        case2 = _create_case(user_id="user1")
+        middle = datetime.utcnow()
+        time.sleep(.01)
```

Comment:
yeah, that's true. these are cases so it would be `server_modified_on`. Not sure it's necessarily worth the effort of figuring out the best way to set those dates during case processing in the text, or alternatively saving each case twice in the test to override it. open to push back on that.